gateway/src/service.rs

```diff
@@ -276,6 +276,22 @@ impl GatewayService {
             .ok_or_else(|| Error::from_kind(ErrorKind::ProjectNotFound))
     }
 
+    pub async fn iter_projects_list(
+        &self,
+    ) -> Result<impl Iterator<Item = (ProjectName, Project)>, Error> {
+        let iter = query("SELECT project_name, project_state FROM projects")
```

Comment:
This is just missing a `WHERE` clause to only return a specific account's projects and not all projects. Might have to rename the method to something like `iter_users_projects` or something :man_shrugging:?

Reply:
@chesedo Thanks for the feedback! I've updated the query to filter by account name and renamed the method.